src/formElementPlugins/ApplicationViewWrapper.tsx

```diff
@@ -165,6 +166,18 @@ const ApplicationViewWrapper: React.FC<ApplicationViewWrapperProps> = (props) =>
 
 export default ApplicationViewWrapper
 
+/* 
+Allows the default value in template to be either an index or string
+value. Number is assumed to be index, else it returns the index of the 
+specified value in the options array. Functions is passed as prop to
+element plug-ins so can be used by any plugin.
+*/
+const getDefaultIndex = (defaultOption: string | number, options: string[]) => {
+  if (typeof defaultOption === 'number') {
+    return defaultOption
+  } else return options.indexOf(defaultOption)
+}
```

Comment:
I think this function would throw an error if you (by mistake on the template data) send a index out-of-range in the `options` array. There should be a safe guard for that.

Reply:
I just checked what happens, and if the returned index number is outside the range, it just has nothing selected, and no run-time error. (The optional chaining operator in `setValue(options?.[getDefaultIndex(defaultOption, options)])` ensures it doesn't try to set a non-existent value.

So having an illegal default value (index or name) behaves the same as having no default value. This seems like the appropriate behaviour don't you think?

Reply:
Yes. That's good